server/coordinator/node_picker_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package coordinator
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/CeresDB/ceresmeta/server/cluster"
+	"github.com/CeresDB/ceresmeta/server/storage"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	NodeLength = 3
+)
+
+func TestRandomNodePicker(t *testing.T) {
```

Comment:
Add some corner cases to test, e.g.:
- empty source nodes;
- non-empty source nodes with no online nodes in them;